docs/concepts/index.rst

```diff
@@ -0,0 +1,263 @@
+      binary quadratic model
+      BQM
+         A collection of binary-valued variables  (variables that can be assigned two values, for example -1, 1) 
+         with associated linear and quadratic biases. Sometimes referred to in other tools as a problem.
+         See a fuller description under :doc:`Binary Quadratic Models </concepts/bqm>`.
+
+      Chain
+         One or more nodes or qubits in a target graph that represent a single variable in
+         the source graph. See :term:`embedding`.
+         See a fuller description under :doc:`Minor-Embedding </concepts/embedding>`.
+
+      Chain length
+         The number of qubits in a :term:`Chain`.
+         See a fuller description under :doc:`Minor-Embedding </concepts/embedding>`.
+
+      Chain strength
+         Magnitude of the negative quadratic bias applied between variables to form a chain.
+         See a fuller description under :doc:`Minor-Embedding </concepts/embedding>`.
+
+      Chimera
+         The D-Wave :term:`QPU` is a lattice of interconnected qubits. While some qubits
+         connect to others via couplers, the D-Wave QPU is not fully connected.
+         Instead, the qubits interconnect in an architecture known as Chimera.
+         See a fuller description under :doc:`QPU Topology </concepts/topology>`.
+
+      Complete graph
+      Fully connected
+          See `complete graph`_. on wikipedia. A fully connected or complete
+          :term:`binary quadratic model` is one that has interactions between
+          all of its variables.
+
+          .. _complete graph: https://en.wikipedia.org/wiki/Complete_graph
+
+      Composed sampler
+          Samplers that apply pre- and/or post-processing to binary quadratic programs without
+          changing the underlying :term:`sampler` implementation by layering composite patterns
+          on the sampler. For example, a composed sampler might add spin transformations when
+          sampling from the D-Wave system.
+
+      Composite
+          A :term:`sampler` can be composed. The
+          `composite pattern <https://en.wikipedia.org/wiki/Composite_pattern>`_
+          allows layers of pre- and post-processing to be applied to binary quadratic
+          programs without needing to change the underlying sampler implementation.
+          We refer to these layers as "composites". A composed sampler includes at least one
+          sampler and possibly many composites.
+
+      Embed
+      Embedding
+      Minor-embed
+      Minor-embedding
+         The nodes and edges on the graph that represents an objective function
+         translate to the qubits and couplers in :term:`Chimera`. Each logical qubit, in
+         the graph of the :term:`objective function`, may be represented by one or more
+         physical qubits. The process of mapping the logical qubits to physical
+         qubits is known as minor embedding.
+         See a fuller description under :doc:`Minor-Embedding </concepts/embedding>`.
+
```

Comment:
`Excited State` would also be useful to be able to link to

Reply:
`Ground state` as well

Reply:
Added both